recipes/solo-epd-loader/LICENSE.rst

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2022, Jan Gieseler
```

Comment:
The license is already included in the source tarball (in `licenses`) so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.